src/views/Player.vue

```diff
@@ -71,12 +71,16 @@
         :questionSetTitle="currentQsetTitle"
         :questionSetStates="questionSetStates"
         :qsetIndexLimits="currentQsetIndexLimits"
-        :quizTimeLimit="quizTimeLimit"
+        :quizTimeLimit="quizTimeLimit?.max"
```

Comment:
wait why sharing only max?

Reply:
acha quiz time limit is supposed to be a number. this is bad naming na. 
in one place, quiz time limit refers to {min, max} limits

in splash and instruction quiztimelimit refers only to max. 

that shouldn't be the case. 

you should take both min and max. and use only max for now in instruction page.

Reply:
Because in instructionPage in Test Overview section, only maximum timelimit is required to show the duration of the test.